eks_infra/create_cluster.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+#/ Usage: 
+#/ export AWS_REGION=<AWS-Region>
+#/ export EC2_KEY_PAIR_NAME=<EC2-Key-Pair-Name>
+#/ ./create_cluster.sh eks_cluster_name eks_version
+
+set -e
+
+# Function to create EC2 key pair
+function create_ec2_key_pair() {
+    aws ec2 create-key-pair \
+    --key-name ${1} \
+    --query 'KeyMaterial' \
+    --output text > ./${1}-KeyPair.pem
+}
+
+# Function to create EKS cluster using eksctl. 
+# The cluster name follows the {framework}-{build_context} convention
+function create_eks_cluster() {
+
+    if [ "${3}" = "us-east-1" ]; then
+      ZONE_LIST=(a b d)
+    else
+      ZONE_LIST=(a b c)
+    fi
+    
+    eksctl create cluster \
+    --name ${1} \
+    --version ${2} \
+    --zones=${3}${ZONE_LIST[0]},${3}${ZONE_LIST[1]},${3}${ZONE_LIST[2]} \
+    --without-nodegroup
+}
+
+# Function to create static and dynamic nodegroups in EKS cluster
+function create_node_group(){
+    # static nodegroup
+    eksctl create nodegroup \
+    --name static-nodegroup-${2/./-} \
+    --cluster ${1} \
+    --node-type m5.large \
```

Comment:
Please define this node type as a variable at the top of this script.

Reply:
done